### PR TITLE
Add static file in sale channel

### PR DESCRIPTION
**Pull request by usudaysingh**

### CODE REVIEW

File: channel.py [hunk 1]
```diff
@@ -38,6 +38,9 @@ class SaleChannel(ModelSQL, ModelView):
     """
     __name__ = 'sale.channel'
 
+    static_file = fields.Many2One(
+        "nereid.static.file", "Static File", required=True, select=True)
```
Comment (priyankarani) [inline channel.py]: move ')' to next line

File: setup.py [hunk 2]
```diff
@@ -84,6 +84,7 @@ def run(self):
 
 MODULE2PREFIX = {
     'product_notebook': 'openlabs',
+    'email_queue': 'fio',
```
Comment (priyankarani) [inline setup.py]: why this ?

File: tryton.cfg [hunk 3]
```diff
@@ -6,6 +6,8 @@ depends:
     sale_price_list
     product_notebook
     stock
+    nereid
+    email_queue
```
Comment (priyankarani) [inline tryton.cfg]: why email_queue?

File: view/sale_channel_form.xml [hunk 4]
```diff
@@ -66,6 +66,10 @@
             <field name="read_users" colspan="6"/>
             <field name="create_users" colspan="6"/>
         </page>
+        <page string="Media" id="media">
+            <label name="static_file" />
+            <field name="static_file" colspan="6"/>
```
Comment (priyankarani) [inline view/sale_channel_form.xml]: colspan="5"

File: setup.py (hunk 2 above)
Comment (usudaysingh) [inline setup.py]: Because it was raisng email_queue dependency error.

Comment (usudaysingh) [inline setup.py]: Because it was not able to set prefix .It was using tyrotnd as prefix and raising error.

Comment (priyankarani) [inline setup.py]: you dont need to add email queue, nereid already has email queue dependency

File: tryton.cfg (hunk 3 above)
Comment (priyankarani) [inline tryton.cfg]: nereid already has email queue dependency. remove this

**Comment by priyankarani** [comment]
LGTM

File: view/sale_channel_form.xml [hunk 5]
```diff
@@ -66,6 +66,10 @@
             <field name="read_users" colspan="6"/>
             <field name="create_users" colspan="6"/>
         </page>
+        <page string="Media" id="media">
```
Comment (prakashpp) [inline view/sale_channel_form.xml]: dont create a tab just for a field

**Pull request closed without merging**